### PR TITLE
Use environment variable to enable SDKs

### PR DESCRIPTION
**Pull request by eszlari**
@refi64 @FakeShemp @p1u3o @johnjelinek

usage:
```
FLATPAK_ENABLE_SDK=golang,dotnet flatpak run com.visualstudio.code
```

### CODE REVIEW

**Comment by p1u3o** [comment]
Does this mean after installing an SDK, someone has to manually override the env variable on the Flatpak to run the enable.sh ?

**Comment by eszlari** [comment]
@p1u3o I added an usage example.

File: code.sh [hunk 1]
```diff
@@ -9,25 +9,12 @@ if [ ! -f ${FIRST_RUN} ]; then
...
-  . /usr/lib/sdk/openjdk8/enable.sh
-fi
-
-if [[ -d /usr/lib/sdk/mono5 ]]; then
-  . /usr/lib/sdk/mono5/enable.sh
-fi
+IFS=',' read -ra SDK <<< "$FLATPAK_ENABLE_SDK"
+for i in "${SDK[@]}"; do
+  if [[ -d /usr/lib/sdk/$i ]]; then
+    . /usr/lib/sdk/$i/enable.sh
```
Comment (gasinvein) [inline code.sh]: Maybe add `/usr/lib/sdk/$i/bin` to `PATH` if `enable.sh` doesn't exist?

**Comment by gasinvein** [comment]
This looks exceptionally useful, great idea @eszlari.

File: code.sh (hunk 1 above)
Comment (eszlari) [reply code.sh]: Is there an extension that doesn't provide a `enable.sh`? I thought this is the standard?

Comment (gasinvein) [reply code.sh]: It's not mandatory. For example, `toolchain-i386` (built alongside with freedesktop-sdk itself) doesn't have it.

Comment (gasinvein) [reply code.sh]:
```suggestion
    if [[ -f enable.sh ]]; then
      . /usr/lib/sdk/$i/enable.sh
    else
      export PATH=$PATH:/usr/lib/sdk/$i/bin
    fi
```

**Pull request merged**